src/test/java/dk/cngroup/lentils/controller/ClientControllerCypherDetailIntegrationTest.java

```diff
@@ -11,14 +11,14 @@
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
```

Comment:
Dotaz: je mezi tímto nějaký rozdíl? Na netu jsem našle jen, že je SpringRunner alias toho SpringJunit4classRunneru

Reply:
Pokud mezi tím není rozdíl, tak jsem pro používat všude SpringRunner jelikož to je čitelnější :100:

Reply:
Dík, já to tam dávala na zkoušku, protože mi s tím SpringRunnerem padaly všechny testy, když jsem tam přidávala do Dependencies ještě nějakou třídu pro načítání a úpravu MultipartFile a s tím SpringJUnit4ClassRunner to běželo, ale pak jsem to vyřešila nějak jinak a tohle tam zůstalo omylem.
Jinak si taky myslím, že je to stejné, jenom SpringRunner je novější.